src/Assets/styleController.ts

```diff
@@ -43,6 +46,7 @@ const themes: IThemes = {
       'font-size': '16px',
       '--main-header-color': '#B85A27',
       '--midBlue-color': '#41528C',
+      '--brightOrange-color': '#DE7F43',
```

Comment:
What do you think about removing this js variable since it's not used anywhere?

Reply:
@CalebPena @ivonne-hernandez  Resolved. Icon colors updated:

![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/c2fa579b-91a2-4165-aa15-c21a8a1a7b8c)